### DISCUSSION TITLE
Localise Notes to nl-NL (Dutch)

### DISCUSSION THREAD

**Discussion by Martien** [observablehq/framework#1853] [Q&A] [answered]
What is the best way to localise [Notes](https://observablehq.com/framework/markdown#notes) to nl-NL (Dutch), in addition to the original English version?

E.g. I’d like to add `.waarschuwing`, which inherits all of `.warning`, but uses `.waarschuwing::before {content: "Waarschuwing";}` in order to use a styled container like `::: waarschuwing`.

My first idea is to create a local copy of [src/style/note.css](https://github.com/observablehq/framework/blob/8efdcf45c00f7cb471deab01f65fe93e1b539d44/src/style/note.css).

Please tell me if there is a more appropriate and evolvable way.

**Comment by mootari** [comment, marked as answer]
You can use the `:lang()` selector  and `lang` attribute: https://developer.mozilla.org/en-US/docs/Web/CSS/:lang
```html
<style>
  .note:lang(de)::before {content: "Hinweis"}
</style>

<div class="note">This is a note.</div>

<div class="note" lang="de">Dies ist ein Hinweis.</div>
```
<img width="661" alt="image" src="https://github.com/user-attachments/assets/fb401ad2-af22-471c-9e34-fc02bf46121a">

**Reply by Martien** [reply]
🎉 Wonderful! 🙏Thank you. Very elegant. Learned something again.

Have a nice day.
